datasets/pit2015.py

```diff
@@ -0,0 +1,65 @@
+import os
+
+import torch
+from torchtext.data.field import Field, RawField
+from torchtext.data.iterator import BucketIterator
+from torchtext.vocab import Vectors
+
+from datasets.castor_dataset import CastorPairDataset
+
+
+class PIT2015(CastorPairDataset):
+    NAME = 'pit2015'
+    NUM_CLASSES = 2
+    ID_FIELD = Field(sequential=False, dtype=torch.float32, use_vocab=False, batch_first=True)
```

Comment:
Maybe use torchtext v0.2.3 for now for the tensor type.

Reply:
Yeah. You are right.